source/includes/kubernetes/_k8_configmaps.md

```diff
@@ -91,17 +91,17 @@ curl -X GET \
 
 Retrieve a configmap and all its info in a given [environment](#administration-environments).
 
-| Attributes                                 | &nbsp;                                                          |
-| ------------------------------------------ | --------------------------------------------------------------- |
-| `id` <br/>_string_                         | The id of the configmap                                        |
-| `apiVersion` <br/>_string_                 | The API version used to retrieve this configmap                |
-| `kind` <br/>_string_                       | The type of the returned resource: ConfigMap                   |
-| `metadata` <br/>_object_                   | The metadata of the configmap                                  |
-| `metadata.creationTimestamp` <br/>_string_ | The date of creation of the configmap as a string              |
-| `metadata.annotations` <br/>_map_          | The annotations associated to the configmag                    |
-| `metadata.name` <br/>_string_              | The name of the configmap                                      |
-| `metadata.namespace` <br/>_string_         | The namespace in which the configmap is created                |
-| `metadata.uid` <br/>_object_               | The UUID of the configmap                                      |
-| `metadata.selfLink` <br/>_object_          | A link uniquely identifying this config map                    |
-| `metadata.resourceVersion` <br/>_object_   | The version returned of the configmap                          |
+| Attributes                                 | &nbsp;                                                  |
+| ------------------------------------------ | ------------------------------------------------------- |
+| `id` <br/>_string_                         | The id of the config map                                |
+| `apiVersion` <br/>_string_                 | The API version used to retrieve this config map        |
+| `kind` <br/>_string_                       | The type of the returned resource. ie, ConfigMap        |
+| `metadata` <br/>_object_                   | The metadata of the config map                          |
+| `metadata.creationTimestamp` <br/>_string_ | The date of creation of the configmap as a string       |
```

Comment:
Looks like this config map needs a space #LeaveRoomForJesus

Reply:
[f17f2a6](https://github.com/cloudops/cloudmc-api-docs/pull/169/commits/f17f2a61c479aca8ff6b7d2313097cc8e72c4a95)